Assets/Scripts/Game/UserInterface/PauseOptionsDropdown.cs

```diff
@@ -152,7 +151,18 @@ public void SetDropdownExpand(bool expand)
 
         #region Private Methods
 
-        void SetBackground(BaseScreenComponent panel, Color color, string textureName)
+        private bool HasApplicableMods()
+        {
```

Comment:
Please pop in a null check here for MidManager.Instance and bounce if null before loop. This can be null when starting from Game scene (e.g. directly loading a save or UI during testing) and will cause an exception if not started from Startup scene.